src/main/java/upb/dice/twitter/MaxIDCheck.java

```diff
@@ -0,0 +1,67 @@
+package upb.dice.twitter;
+
+import twitter4j.GeoLocation;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Retrieves the maxID for a specific query
+ */
+public class MaxIDCheck {
```

Comment:
Please extend the description of this method since it is important to understand that it has to go through the whole file.